Apps/AdminWebClient/.gitattributes

```diff
@@ -0,0 +1,65 @@
+###############################################################################
```

Comment:
I believe the .gitattributes at the root level will suffice and this would attempt to override which isn't our intention.